tests/ui/rust-2018/async-ident.rs

```diff
@@ -11,14 +11,12 @@ macro_rules! foo {
     ($foo:ident) => {};
     ($async:expr, async) => {};
```

Comment:
It's kinda strange how we still deny the plain `async` keyword, but I don't think this needs fixing at this point.